contacts-app/src/components/Button/Button.jsx

```diff
@@ -1,22 +1,32 @@
 import React from "react";
 import styles from "./Button.module.css";
 import classNames from "classnames";
+import PropTypes from "prop-types";
 
 class Button extends React.Component {
   render() {
-    const type = this.props.type;
+    const buttonType = this.props.buttonType;
     const text = this.props.text;
+    const type = this.props.type;
     return (
       <button
+        type={type}
         onClick={event => {
           this.props.handleClick(event);
         }}
-        className={classNames(styles.button, styles[type])}
+        name={text}
+        className={classNames(styles.button, styles[buttonType])}
       >
         {text}
       </button>
     );
   }
 }
 
+Button.propTypes = {
+  buttonType: PropTypes.string,
+  text: PropTypes.string,
```

Comment:
you can also use proptypes to use default values. eg what should `text` be when a use passes in nothing? Or should a use always pass in something?

Reply:
That is a good point, though I thought about the 2 scenarios where the button is used in the app, and cannot think of a good default value for button because the text actually explains different function of the button.
Any suggestion?